### DISCUSSION TITLE
Victron vedirect USB

### DISCUSSION THREAD

**Discussion by blacktoli** [bareboat-necessities/lysmarine_gen#320] [Q&A]
Hello,

Since I installed the latest image, I am unable to start the vedirect-serial saying (Failed to start: Delimiter is not a constructor)

The serial port is: /dev/serial/by-id/usb-VictronEnergy_BV_VE_Direct_cable_VE5FF61F-if00-port0 but I also tried with port /dev/ttyUSB2

Any ideas why I cannot start the pluggin?

Thanks

**Comment by blacktoli** [comment]
Forgot to mention that when I debug, it shows the data correctly received from my mppt

**Comment by mgrouch** [comment]
You should be able to fix it if it gives line number. Or you can check with a developer of that signalK plugin to fix it.

https://github.com/SignalK/vedirect-serial-usb

**Reply by blacktoli** [reply]
This plugin was not updated since 2 years. No one uses this plugin? I don't think it worth it installing another RPI for the VenusOS

**Comment by mgrouch** [comment]
Old code broke against updated nodeJS libraries.
Old code needs to be fixed. Look for word 'Delimiter' in failing module. I think it's easy enough to fix yourself.
Or ask on signalK slack channel.

Thanks

**Comment by mgrouch** [comment]
Issue is fixed

https://github.com/SignalK/vedirect-serial-usb/issues/49
